src/intl/en/learn-quizzes.json

```diff
@@ -324,5 +324,75 @@
+  "j001-b-label": "Penalty for being offline, validator is immediately prohibited from attesting ever again",
+  "j001-b-explanation": "Being offline does NOT result in slashing. While slashing will result in the validator being prohibited from ever attesting again and is ultimately forcefully ejected, being offline will NOT result in ejection from the network.",
+  "j001-c-label": "Penalty for breaking specific consensus rules, rewards resume after slashing",
+  "j001-c-explanation": "Slashing is a serious penalty for breaking specific consensus rules that present a threat to the network. As such, once a validator is slashed they are immediately prohibited from attesting any further, and are ultimately forcefully ejected from the network and remaining ETH is withdrawn to the owner.",
+  "j001-d-label": "Penalty for breaking specific consensus rules, validator is immediately prohibited from attesting ever again",
+  "j001-d-explanation": "Slashing is a serious penalty for breaking specific consensus rules that present a threat to the network. As such, once a validator is slashed they are immediately prohibited from attesting any further, and are ultimately forcefully ejected from the network and remaining ETH is withdrawn to the owner.",
+  "j002-prompt": "What happens if a validator goes offline?",
+  "j002-a-label": "No affect on rewards",
+  "j002-a-explanation": "Penalties are incurred when a validator is unavailable to attest to the state of the chain for any given epoch. The size of these penalties is approximately equal to 75% of what the reward for a proper attestation would have been. Rewards resume when validator goes back online, and NO slashing occurs.",
+  "j002-b-label": "Inactivity penalties are incurred only while unavailable",
+  "j002-b-explanation": "While unavailable, a validator will incur small inactivity penalties, approximately equal to 75% of what the reward would have been for a proper attestation. In rare/extreme cases where the network is not finalizing (i.e. over 1/3 of the network is also offline), these penalties are significantly greater. Rewards resume when validator goes back online, and no slashing occurs.",
+  "j002-c-label": "Immediate slashing and removal from the network",
+  "j002-c-explanation": "This is a common misconception, but going offline does NOT result in slashing! Slashing is a specific type of penalty for more serious offense, with larger penalties and also results in removal from the validator set.",
+  "j002-d-label": "One week delay before slashing and ejection",
+  "j002-d-explanation": "Being offline does NOT result in slashing, even after extended period of time. A validator could theoretically be offline for years without being slashed, though inactivity penalties would mount if the validator does not exit.",
+  "j003-prompt": "What is the max effective balance of a validator?",
```

Comment:
As far as I can tell, this question isn't really answered on the page explicitly.

Reply:
I'll just remove this one from the quiz array at the very least... can find another home for it, or update the content to include this information, or just remove it.